### PR TITLE
SC-2 Adding Structure for Screencast

### PR DESCRIPTION
**Pull request by siba2893**
- Adding basic structure for javascript files for screencast
- Adding needed gems for Screencast
- Moving inline CSS to a CSS file.

### CODE REVIEW

**Comment by RoyRivera** [comment]
LG :horse_racing: just use the correct gem git path, instead of the local one.

File: app/assets/stylesheets/screencast.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.remove_padding{padding: 0;}
+.remove_margin_top{margin-top: 0;}
+.remove_border_radius_nav{border-radius: 0;}
```
Comment (andresgap) [inline app/assets/stylesheets/screencast.scss]: Configure end of file lines in your IDE

File: app/assets/stylesheets/screencast.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.remove_padding{padding: 0;}
```
Comment (andresgap) [inline app/assets/stylesheets/screencast.scss]: Use styles formatting

File: Gemfile [hunk 3]
```diff
@@ -33,10 +35,12 @@ gem 'bootstrap-sass', '~> 3.3.6'
 
 # Engines
 gem 'rop', git: 'git@github.com:rails-gap/rop.git', branch: 'develop'
-gem 'screencasts', git: 'git@github.com:rails-gap/screencasts.git', branch: 'develop'
+#gem 'screencasts', git: 'git@github.com:rails-gap/screencasts.git', branch: 'develop'
+gem 'screencasts', :path => '../screencasts/'
```
Comment (andresgap) [inline Gemfile]: Use development or leave it using the git link

**Pull request merged**